src/eventHandlers/nftOperations.ts

```diff
@@ -66,3 +66,15 @@ export const nftOperationEntityHandler = async (
   }
   await nftOperationRecord.save()
 }
+
+export enum NFTOperation {
```

Comment:
We should use lower case for values and also add additional auction's operations: 

```
export enum NFTOperation {
  Create = "create",
  Burn = "burn",
  Transfer = "transfer",
  Delegate = "delegate",
  Undelegate = "undelegate",
  SetRoyalty = "setRoyalty",
  Sell = "sell",
  List = "list",
  Unlist = "unlist",
  CompleteAuction = "completeAuction",
  BuyItNowAuction = "buyItNowAuction",
  CreateAuction = "createAuction",
  AddBid = "addBid",
  RemoveBid = "removeBid",
}
```